src/VirtoAppLayout/VirtoAppLayout.tsx

```diff
@@ -83,7 +77,7 @@ function Layout({
         </Box>
         <Box
           sx={{
-            background: theme => theme.palette.background.default,
+            background: `var(--ipg-palette-background-paper)`,
```

Comment:
Why hard code a css variable here. Why aren't we getting it from the theme? What if MUI change the css variable name in the future?

Reply:
I added this purely for an example to show how we can use `var(...)` to access theme color in UI. No objection for using `theme => theme.palette.background.default`. 

The prefix `--ipg` is added by us. The css variable you mean is this `palette-background-paper` part?